yarn-project/end-to-end/src/e2e_public_token_contract.test.ts

```diff
@@ -95,7 +104,12 @@ describe('e2e_public_token_contract', () => {
     expect(receipts.map(r => r.status)).toEqual(times(3, () => TxStatus.MINED));
     expect(receipts.map(r => r.blockNumber)).toEqual(times(3, () => receipts[0].blockNumber));
 
-    await expectAztecStorageSlot(logger, aztecRpcServer, contract, balanceSlot, recipient.toField(), mintAmount * 3n);
+    const balance = await cc.l2.loadPublic(
```

Comment:
given how often this is used, we can probably make another helper like `getPublicBalanceOf(contractAddress, balanceSlot, recipient)` that calls this